src/browser/netsurf.zig

```diff
@@ -2139,13 +2156,32 @@ pub inline fn documentCreateDocument(title: ?[]const u8) !*DocumentHTML {
     return doc_html;
 }
 
+pub inline fn documentCreateHTMLElement(doc: *Document, tag_name: []const u8) !*Element {
```

Comment:
I know everything in netsurf.zig is inlined, but Andrew's not a fan:

https://ziggit.dev/t/inlining-functions/1341/3